tests/wea_test.py

```diff
@@ -65,8 +92,37 @@ def test_import_stat(self):
         assert wea_from_json.diffuse_horizontal_radiation.values == \
             wea_from_stat.diffuse_horizontal_radiation.values
 
+    def test_global_horizontal(self):
+        """Test the global horizontal radiation on method."""
+        stat_path = './tests/stat/chicago.stat'
+        wea_from_stat = Wea.from_stat_file(stat_path)
+
+        diffuse_horiz_rad = wea_from_stat.diffuse_horizontal_radiation
+        direct_horiz_rad = wea_from_stat.direct_horizontal_radiation
+        glob_horiz_rad = wea_from_stat.global_horizontal_radiation
+
+        assert [x.value for x in glob_horiz_rad] == pytest.approx(
+            [x + y for x, y in zip(diffuse_horiz_rad, direct_horiz_rad)], rel=1e-3)
+
     def test_radiation_on_surface(self):
```

Comment:
This comment is unrelated to test PR.

I think we don't want to use term `surface` under the wea class. It can be confusing to have radiation calculation on a surface under Wea class. Think about someone searching the API for keyword radiation and finds this. It can be misleading. I would suggest to rename this method to `radiation_at_angle` or something like that.